main.ts

```diff
@@ -85,9 +82,81 @@ class SettingTab extends PluginSettingTab {
 					.setPlaceholder("Enter your API key")
 					.setValue(this.plugin.settings.openAIApiKey)
 					.onChange(async (value) => {
-						this.plugin.settings.openAIApiKey = value;
-						await this.plugin.saveSettings();
+						await this.plugin.updateSettings({openAIApiKey: value});
+					}),
+			);
+		new Setting(containerEl)
+			.setName("Minimum Summary Size")
+			.setDesc("Minimum number key points to include in generated summary.")
```

Comment:
I think the description here is a bit misleading. For long videos, what this plugin does is to split the transcript into chunks, gets the keypoints from each and then merges the results. 

Here the summary size you mention is the key points count for a chunk, but not for the merged results. It is worth adding one more sentence to the description to explain this.

Reply:
Hi @ozdemir08 , I didn't realize the chunking aspect of the processing -- I thought it was all sent in one request.

How does this description sound:

`Minimum number of key points per video chunk to include in the generated summary. Note: The plugin divides long videos into chunks, extracts key points from each, and then combines them for the final summary.`